settings.gradle

```diff
@@ -37,5 +37,14 @@ file('modules').eachDir { dir ->
     project(":${dir.name}").projectDir = dir
 }
 
+file('examples').eachDir { dir ->
+    if (dir.name == 'build') {
+        return
+    }
+
+    include "examples:${dir.name}"
+    project(":examples:${dir.name}").projectDir = dir
```

Comment:
Treating each example as subproject might become a little issue for scalability (Gradle's configuration time) in the long run. Imagine you have hundreds of example projects here. Probably won't happen very soon but is something to consider. I'd use TestKit here and execute them in parallel to speed up the build.

Happy to send a PR once you merged this one.

Reply:
That's a great idea! I'll try to get this merged soon without too much changes so we have finally everything on Gradle and then we can work on improving it :slightly_smiling_face:

Reply:
The composite builds should solve this as well :)